server/views/layout.njk

```diff
@@ -29,12 +29,14 @@
 {% endblock %}
 
 {% block beforeContent %}
+  <nav>
```

Comment:
The phase tag isn't really a navbar, can we include it in the `header.njk` instead?

Reply:
yes, I'll take a look at that first thing.

Reply:
We already have a role of `banner` in the main title bar, and can't have two of these, so made this an aria role of `complementary` with a div id of `phase-banner`.